extensions/gitpod-web/src/extension.ts

```diff
@@ -577,6 +578,27 @@ function registerPorts(context: GitpodExtensionContext): void {
 			}
 		});
 	}
+
+	const packageJSON = context.extension.packageJSON;
+	const experiments = new ExperimentalSettings(packageJSON.configcatKey, packageJSON.version, context.logger);
+	context.subscriptions.push(experiments);
+
+	const isSaaSGitpod = context.info.getGitpodHost() === 'https://gitpod.io';
+	async function getPortsViewExperimentEnable(): Promise<boolean> {
+		const isEnabled = isSaaSGitpod
+			? (await experiments.get<boolean>('gitpod.experimental.portsView.enabled', (await context.user).id))!
```

Comment:
Can we add team_id or team_ids as for isUsageBasedBillingEnabled? That we ensure that in our first experiment step we force Gitpod team users to try for one day. There is Server API for it: https://github.com/gitpod-io/gitpod/blob/4d7695b76c48855cccae3bc03d964476a269d294/components/gitpod-protocol/src/gitpod-service.ts#L165 Not sure whether one time token has access to it though. Maybe webapp team deployment will be needed.